src/index.ts

```diff
@@ -100,6 +100,16 @@ const deploymentBlockOption = new Option(
   .env("DEPLOYMENT_BLOCK")
   .argParser(parseIntOption);
 
+const multiOrderBookApiOption = new Option(
+  "--orderBookApi <orderBookApi...>",
+  "Orderbook API base URLs (i.e. https://api.cow.fi/mainnet, https://api.cow.fi/xdai, etc.)"
+).default([]);
+
+const orderBookApiOption = new Option(
+  "--orderBookApi <orderBookApi>",
+  "Orderbook API base URL (i.e. https://api.cow.fi/mainnet)"
+).default(undefined);
```

Comment:
As this option doesn't have "make mandatory", is setting the default to `undefined` required?

Reply:
https://github.com/cowprotocol/watch-tower/pull/106#event-10687336475